UnitTest/UnitTestTests/StringExtensionTests.cs

```diff
@@ -0,0 +1,51 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using UnitTest;
+
+namespace UnitTestTests
+{
+    [TestClass]
+    public class StringExtensionTests
+    {
+        [TestMethod]
+        public void Handle_When_ColorIsBlack_Then_MustBeTrue()
+        {
+            // Arrange
+            const string color = "black";
+
+            // Act
+            bool actual = color.IsBaseColor();
+
+            // Assert
+            const bool expected = true;
```

Comment:
this one should be in arrange section, all setup of the test and expected results should be in arrange

Reply:
Done in 3b0d4f72e4804b22ea2b3b5437831f9e099c7dd2